callmetadata.schema.json

```diff
@@ -5,29 +5,87 @@
+                    "example": 27978
+                }
+            }
+        },
+        "member": {
+            "name": {
+                "type": "string",
+                "example": "Steve Jones",
+                "description": "Name of the participant communicating as defined in schema.org"
+            },
+            "groupName": {
+                "type": "string",
+                "example": "Energy Desk",
+                "description": "Name of the group for which the participant is representing in the communication"
+            },
+            "firmName": {
+                "type": "string",
+                "example": "XYZ Firm",
+                "description": "Name of the participant's firm represnted in the communication"
+            },
+            "timeZoneOffset": {
+                "type": "string",
+                "example": "UTC -05:00:00",
+                "description": "The UTC timezone offset for the participant"
+            }
+        },
+        "participant": {
+            "type": "object",
+            "description": "Specific details about a participant in a given communication",
+            "properties": {
+                "member": {
+                    "$ref": "#/definitions/member"
                 },
-                "groupName": {
+                "involvement": {
+                    "type": "string",
+                    "description": "How the participant joined the communication. Did they initiate it, receieve a request or join an ongoing communication?",
+                    "enum": [
+                        "source",
+                        "destination",
+                        "participant"
+                    ]
+                },
+                "address": {
                     "type": "string",
-                    "example": "Energy Desk",
-                    "description": "Name of the farEnd group for the call. It may changed by the farEnd Firm"
+                    "example": "sip:9991234567@voip-provider.net",
+                    "description": "The address of the participant. It can be a URL, a username or any other type of address used to reach this participant."
                 },
-                "farEndName": {
+                "joinTime": {
                     "type": "string",
-                    "example": "Steve Jones",
-                    "description": "The name (first and last) of the farEnd User on the call"
+                    "format": "date-time",
+                    "example": "2016-06-18T07:50:30Z",
+                    "description": "Date and time when the participant joined the conversation"
+                },
+                "exitTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "example": "2016-06-18T08:15:15Z",
+                    "description": "Date and time when the participant exited the conversation"
+                },
+                "states": {
```

Comment:
@pinczo I suggest to change the subject of this PR to this `states` update, and move other proposals to the participants PR https://github.com/finos/voice-metadata-standard/pull/44/. Since I've already updated the PR with the `involvement` change, I think we just need to discuss and agree/disagree with the `member` update.